Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -1766,7 +1795,8 @@ def __init__(self, parent=None, orientation=Qt.Vertical, value=0.0,
         super().__init__(parent, **kwargs)
 
         self.setAcceptedMouseButtons(Qt.LeftButton)
-        self.setPen(make_pen(brush=QColor(50, 50, 50), width=1, cosmetic=False))
+        self.setPen(make_pen(brush=QColor(50, 50, 50), width=1, cosmetic=False,
+                             style=Qt.DashLine))
```

Comment:
Cut-off line often crosses the new labels, which looked ugly. Dashed line looks better.